### ISSUE TITLE
Docker image versioning

### ISSUE THREAD

**Issue by Filipoliko** [joelgriffith/browserless#86]
Hi,
i would like to propose a change of the current docker image versioning.

## WHY?
In the latest release, you introduced a change of fonts in the docker image. We are using the image for visual regression testing and it broke our pipelines. We had to build the older version of your image and use that one to fix the problem for everybody, who did depend on this image, without having to update most of the reference images. My personal opinion is, that once an image is released, it should not really change (with some exceptions like latest versions).

We still love your image and want to use it, but the current way of versioning is not risk free for us.

## HOW?
Lets change the versioning from `browserless/$(browser):$(browser-version)` to `browserless/$(browser)-$(browser-version):$(browserless-version)`. This way, we would have a new docker image(s) with each browserless release, while not changing the already released images. So when a breaking change is introduced, it will not affect the older versions.

**Examples:**
For the release of puppeteer version 1.9.0 and browserless version 1.0.0, I suggest to create docker image tagged as `browserless/puppeteer-1.9.0:1.0.0`.

For the release of chrome version 71 and browserless version 1.0.0, I suggest to create docker image tagged as `browserless/chrome-71:1.0.0`.

There probably should be also a latest browserless version for each image, like this: `browserless/chrome-71:latest`.

_Maybe chrome should have only stable version, like `browserless/chrome-stable:1.0.0`?_

There should be probably some rules as when to end support of some older puppeteer/chrome versions so you would not have to release hundreds of docker images with each browserless release.

What do you think about this?

Thanks for your work!

**Comment by joelgriffith** [comment]
Thanks for putting this forward — totally agree. I might change the order of the numbers so that the browserless version comes before the puppeteer/chrome version.

I’m hopeful that GitHubs automation tool will help with this since there’s a lot of code in this repo for doing docker releases, and once that is in GA I’ll look at moving over to this new versioning semantic!

**Comment by joelgriffith** [comment]
Yup, this is hot on my radar. Unforunately I'm the blocker as I need to do some tweaks to other systems, but I should have something ready here in the coming days

**Comment by joelgriffith** [comment]
https://github.com/joelgriffith/browserless/pull/115/files now has some updates to make this happen + a Changelog to go alongside releases. Currently releases are done locally when things are stable. It's not super ideal at this point, but we'll eventually land on something like Actions if it can support the odd multi-version releases we do.
